gemd/entity/object/has_template_check_generator.py

```diff
@@ -21,7 +21,7 @@ def template(self) -> Union[BaseTemplate, LinkByUID]:
 
     def _generate_template_check(self,
                                  validate: Callable[["HasTemplateCheckGenerator", T], bool]
-                                 ) -> Callable[[T], None]:
+                                 ) -> Callable[[T], Any]:
```

Comment:
Why `Any`, especially since the doc string says it returns a function from `Attribute` to `None`?

Reply:
Nice catch; I got too aggressive in copying the `validate_list` signature up.